pynestml/codegeneration/resources_nest/point_neuron/common/SynapseHeader.h.jinja2

```diff
@@ -335,67 +389,13 @@ private:
 {%- endfilter %}
 
   /**
-  * Free parameters of the synapse.
-  *
-  {{synapse.print_parameter_comment("*")}}
-  *
-  * These are the parameters that can be set by the user through @c SetStatus.
-  * They are initialized from the model prototype when the node is created.
-  * Parameters do not change during calls to @c update() and are not reset by
-  * @c ResetNetwork.
-  *
-  * @note Parameters_ need neither copy constructor nor @c operator=(), since
-  *       all its members are copied properly by the default copy constructor
-  *       and assignment operator. Important:
-  *       - If Parameters_ contained @c Time members, you need to define the
-  *         assignment operator to recalibrate all members of type @c Time . You
-  *         may also want to define the assignment operator.
-  *       - If Parameters_ contained members that cannot copy themselves, such
-  *         as C-style arrays, you need to define the copy constructor and
-  *         assignment operator to copy those members.
-  */
-  struct Parameters_{
-{%- filter indent(4,True) %}
-{%- for variable in synapse.get_parameter_symbols() %}
-{%-     set isHomogeneous = PyNestMLLexer["DECORATOR_HOMOGENEOUS"] in variable.get_decorators() %}
-{%-     if (not isHomogeneous) %}
-{%-   include 'directives/MemberDeclaration.jinja2' %}
-{%-     else %}
-    // N.B. the parameter `{{names.name(variable)}}` is defined in the common properties class
-{%-     endif %}
-{%- endfor %}
-{%- endfilter %}
-
-{% if uses_numeric_solver %}
-    double __gsl_error_tol;
-{% endif %}
-
-    /** Initialize parameters to their default values. */
-    Parameters_() {};
-  };
+   * Update internal state (``S_``) of the synapse according to the dynamical equations defined in the model
```

Comment:
Could you also update the comment that it also includes the statements from the `update` block?